04-onion-routing.md

```diff
@@ -1407,6 +1438,88 @@ The _origin node_:
   - MAY use the data specified in the various failure types for debugging
   purposes.
 
+## Recommendations for Reputation
```

Comment:
This section is helpful, makes it much more concrete what to think of when talking about a reputation system in the context of a routing node.